packages/web-react/src/components/Dropdown/README.md

```diff
@@ -39,6 +39,62 @@ import { Dropdown } from '@lmc-eu/spirit-web-react/components';
 | `trigger['aria-controls']` | `string`                                           | Trigger aria controls   |
 | `trigger.ref`              | `LegacyRef<HTMLButtonElement & HTMLAnchorElement>` | Trigger reference       |
 
+---
+
+# DropdownModern
```

Comment:
Missing deprecation message about that this component will be renamed to `Dropdown` in the future.

Reply:
message added